src/pages/AddressInfoPage/AddressTransactionRow.tsx

```diff
@@ -111,72 +113,111 @@ const AddressTransactionRow: FC<AddressTransactionRowProps> = ({ transaction: t,
-                      outputs={t.outputs}
-                      IOItemWrapper={IODetailsContainer}
-                      addressMaxWidth="180px"
-                      flex
-                    />
-                  )}
-                </IODetailList>
-              </TableRow>
-            </TableBody>
-          </Table>
-        </AnimatedCell>
-      </TableDetailsRow>
+        {!isPending && (infoType === 'move' || infoType === 'out' ? renderOutputAccounts() : renderInputAccounts())}
+        {!isPending && (
+          <AmountCell>
+            {assets.map(({ id, amount, symbol, decimals }) => (
+              <Amount
+                key={id}
+                value={amount}
+                highlight={!isMoved}
+                showPlusMinus={!isMoved}
+                color={isMoved ? theme.font.secondary : undefined}
+                suffix={symbol}
+                decimals={decimals}
+                isUnknownToken={!symbol}
+              />
+            ))}
+          </AmountCell>
+        )}
+        {!isPending && <DetailToggle isOpen={detailOpen} />}
+      </TableRowStyled>
+      {!isPending && (
+        <TableDetailsRow openCondition={detailOpen}>
+          <AnimatedCell colSpan={7}>
+            <Table>
+              <TableHeader headerTitles={['Inputs', '', 'Outputs']} columnWidths={['', '50px', '']} compact />
+              <TableBody>
+                <TableRow>
+                  <IODetailList>
+                    {t.inputs && t.inputs.length > 0 ? (
+                      <TransactionIOList
+                        inputs={t.inputs}
+                        IOItemWrapper={IODetailsContainer}
+                        addressMaxWidth="180px"
+                        flex
+                      />
+                    ) : (
+                      <BlockRewardInputLabel>Block rewards</BlockRewardInputLabel>
+                    )}
+                  </IODetailList>
+
+                  <span style={{ textAlign: 'center' }}>
+                    <ArrowRight size={12} />
+                  </span>
+
+                  <IODetailList>
+                    {t.outputs && (
+                      <TransactionIOList
+                        outputs={t.outputs}
+                        IOItemWrapper={IODetailsContainer}
+                        addressMaxWidth="180px"
+                        flex
+                      />
+                    )}
+                  </IODetailList>
+                </TableRow>
+              </TableBody>
+            </Table>
+          </AnimatedCell>
+        </TableDetailsRow>
+      )}
     </>
   )
 }
 
 export default AddressTransactionRow
 
+const TableRowStyled = styled(TableRow)<{ pending: boolean }>`
+  ${({ pending, theme }) =>
+    pending &&
+    css`
+      > * {
+        opacity: 0.5;
+        animation: opacity-breathing 2s ease infinite;
+      }
+      background-color: ${theme.bg.secondary};
+      border-bottom: 1px solid ${theme.border.secondary};
```

Comment:
super mega nit: I would give priority to the styles of the current component than to the styles of its children:

```suggestion
      background-color: ${theme.bg.secondary};
      border-bottom: 1px solid ${theme.border.secondary};
      
      > * {
        opacity: 0.5;
        animation: opacity-breathing 2s ease infinite;
      }
```